app/serializers/npq_participant_serializer.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require "jsonapi/serializer/instrumentation"
+
+class NPQParticipantSerializer
+  include JSONAPI::Serializer
+  include JSONAPI::Serializer::Instrumentation
+
+  set_id :id
+  set_type :'npq-participant'
+
+  attributes :participant_id#, :date_of_birth
```

Comment:
Let's delete it rather than commenting it out

Reply:
Also, can we add email and full name here? It should hopefully help cross-reference the participants to providers.